From 4ee984e9391568d679d62013157c30d1a14aecd8 Mon Sep 17 00:00:00 2001
From: Douglas Freed <dwfreed@mtu.edu>
Date: Mon, 23 Dec 2013 08:43:10 +0000
Subject: [libnftables PATCH] build: properly handle --without-{xml,json}-parsing

This patch fixes how --without-{xml,json}-parsing (and subsequently
--with-{xml,json}-parsing=no) is handled.  Prior to this,
--without-{xml,json}-parsing actually resulted in libnftables being
built with that parsing enabled.

Signed-off-by: Douglas Freed <dwfreed@mtu.edu>
---
 configure.ac | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index b023755..6219737 100644
--- a/configure.ac
+++ b/configure.ac
@@ -16,11 +16,9 @@ m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 dnl Dependencies
 PKG_CHECK_MODULES([LIBMNL], [libmnl >= 1.0.0])
 AC_ARG_WITH([xml-parsing],
-	AS_HELP_STRING([--with-xml-parsing], [XML parsing support]),
-	[with_xml_parsing="yes"], [with_xml_parsing="no"])
+	AS_HELP_STRING([--with-xml-parsing], [XML parsing support]))
 AC_ARG_WITH([json-parsing],
-	AS_HELP_STRING([--with-json-parsing], [JSON parsing support]),
-	[with_json_parsing="yes"], [with_json_parsing="no"])
+	AS_HELP_STRING([--with-json-parsing], [JSON parsing support]))
 
 AS_IF([test "x$with_xml_parsing" = "xyes"], [
 	PKG_CHECK_MODULES([LIBXML], [mxml >= 2.6])
-- 
1.8.5.2

